src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -1253,6 +1253,10 @@ MavlinkReceiver::handle_message_vision_position_estimate(mavlink_message_t *msg)
 	vision_position.y = pos.y;
 	vision_position.z = pos.z;
 
+	//copy horizontal and vertical covariances
+	vision_position.eph = fmaxf(pos.covariance[0], pos.covariance[6]);
```

Comment:
Why?

Reply:
what do you mean why?

Reply:
Why is eph the max of covariance index 0 and covariance index 6?
Why is epv the covariance at index 11?
Why are these magic numbers not defined anywhere?

Reply:
Those are just the indices of the diagonal terms for position, but yeah we should document/define them somewhere.

Reply:
Yes I know, just trying to illustrate why magic numbers generally aren't good thing.